API_GUIDE.md

```diff
@@ -152,6 +160,25 @@ See the
 [full multiprocessing example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist.py)
 for more on training a network on multiple XLA devices with multi-processing.
 
+### Running on Multiple XLA Hosts
+Multi-host setup for different accelerators can be very different. This doc will talk about the device independent bits of multi-host training and will use the TPU + PJRT runtime(currently available on 1.13 and 2.x releases) as an example.
```

Comment:
Maybe add a one sentence description of what PJRT is and does.
Are there any changes in the earlier training code that are required to get it running on multiple hosts?

Reply:
I stated those(no code change, where to find PJRT doc) at the end of this section, do you think it is better to talk about hose in the beginning?

Reply:
My bad, must have missed that. It's good where it is.